[2024-07-22T02:31:39.271+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-04-23T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:31:39.289+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-04-23T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:31:39.290+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:31:39.293+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:31:39.297+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:31:39.338+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-04-23 00:00:00+00:00
[2024-07-22T02:31:39.355+0000] {standard_task_runner.py:55} INFO - Started process 935 to run task
[2024-07-22T02:31:39.372+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-04-23T00:00:00+00:00', '--job-id', '215', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpsj6cb7p9', '--map-index', '1']
[2024-07-22T02:31:39.378+0000] {standard_task_runner.py:83} INFO - Job 215: Subtask add_one
[2024-07-22T02:31:39.617+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 215 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-04-23T00:00:00+00:00/task_id=add_one/map_index=1/attempt=1.log'; 935)
[2024-07-22T02:31:39.646+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:31:39.697+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
